scripts/bootstrap.sh

```diff
@@ -45,15 +54,10 @@ ExecStop=-/usr/bin/docker stop ${container_name}
 WantedBy=multi-user.target
 CONF
 
+systemctl daemon-reload
+
+
 systemctl enable /etc/systemd/system/conjur.service
```

Comment:
Can we change this to `dap.service`?

Reply:
That out of scope for this PR. We're doing the OS change only and keeping the other PR open for further changes/improvements.

Reply:
If changing this is super involved, I'm fine changing it at a future point.  It just feels like a good time to remove the term `Conjur` when talking about our enterprise product.